static/src/javascripts-legacy/projects/common/views/svgs.js

```diff
@@ -145,10 +143,11 @@ define([
     };
 
     return function (name, classes, title) {
-        if (config.switches.webpack) {
+        // #wp-rjs - needed by karma
+        if (svgs[name].markup) { // webpack way
```

Comment:
yeah i think this could do with looking at. but fwiw, the reason those classes were added by default initially was to mimic the behaviour of the play helper, just to make sure you didn't get any nasty surprises